ujson.go

```diff
@@ -6,17 +6,20 @@ import (
 )
 
 type JSON struct {
-	root interface{}
+	Root interface{}
```

Comment:
Since the Array api returns an []interface{}, we can pass that as the Root and then use the existing api to extract the correct typed value.

Example:
```go
raw := []byte(`{"data":[2,5,7]}`)
decoded, _ := ujson.NewFromBytes(raw)
data := decoded.Get("data").Array()
for _, val := range *data{
    fmt.Printf("Before: %#v\t", val)
    fmt.Printf("After: %#v\n", (&ujson.JSON{Root: val}).Int64(0))
}
```
Output:
```
Before: ujson.numeric{0x32}	After: 2
Before: ujson.numeric{0x35}	After: 5
Before: ujson.numeric{0x37}	After: 7
```